src/univariate/continuous/skewexponentialpower.jl

```diff
@@ -0,0 +1,115 @@
+"""
+    SkewExponentialPower(μ, σ, p, α)
```

Comment:
Is there a reason why you called it `SkewExponentialPower` instead of e.g. `SkewedExponentialPower`?

Reply:
Thanks for all the comments! I will adjust asap. Named it SkewedExponentialPower first but noticed the skewnormal.jl naming and changed skewed to skew. But I'll revert to skewed as that is the term used in the paper :)